torch/nn/modules/module.py

```diff
@@ -1313,8 +1313,12 @@ def requires_grad_(self: T, requires_grad: bool = True) -> T:
             p.requires_grad_(requires_grad)
         return self
 
-    def zero_grad(self) -> None:
-        r"""Sets gradients of all model parameters to zero."""
+    def zero_grad(self, set_to_none=False) -> None:
+        r"""Sets gradients of all model parameters to zero.
+
+        Arguments:
+            set_to_none (bool): instead of setting to zero, set the grad to None.
```

Comment:
Can you link to the `nn.optim` version of this doc that contains more details about the change of behavior?

Reply:
`torch.optim`, right?

Reply:
Yes, typo!

Reply:
```
See :func:`torch.optim.optimizer.zero_grad` for details.
```